src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequestHelper.cs

```diff
@@ -16,10 +16,21 @@ internal static class SilentRequestHelper
     {
         internal static async Task<MsalTokenResponse> RefreshAccessTokenAsync(MsalRefreshTokenCacheItem msalRefreshTokenItem, RequestBase request, AuthenticationRequestParameters authenticationRequestParameters, CancellationToken cancellationToken)
         {
+            const string MAMEnrollmentIdKey = "microsoft_enrollment_id";
             authenticationRequestParameters.RequestContext.Logger.Verbose("Refreshing access token...");
             await authenticationRequestParameters.AuthorityManager.RunInstanceDiscoveryAndValidationAsync().ConfigureAwait(false);
 
-            var msalTokenResponse = await request.SendTokenRequestAsync(GetBodyParameters(msalRefreshTokenItem.Secret), cancellationToken)
+            var dict = GetBodyParameters(msalRefreshTokenItem.Secret);
+
+            var enrollmentIDs = request.ServiceBundle.PlatformProxy.GetEnrollmentIds();
```

Comment:
Is it worth adding this `GetEnrollmentIds` in the platform proxy if only iOS implements them/

Reply:
SilentRequesthelper is in the common code. So it calls the platform proxy.

Reply:
Added as #if iOS